decoder.rb

```diff
@@ -0,0 +1,19 @@
+def code_hash(key)
+  code = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+    '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L',
+    '--' => 'M', '-.' => 'N', '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S',
+    '-' => 'T', '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z'
+  }
+  code[key]
+end
+
+def decode(morse_code)
+  morse_code.split('   ').each do |word|
+    word.split.each do |char|
+      print code_hash(char)
+    end
+    print ' '
+  end
+end
```

Comment:
- [ ] This method takes the sentence and deciphers the sentence immediately. It works well but doesn't meet the requirements. The requirements state that:
- The team should create a method that translates a morse code character into an alphabet. It was implemented✔️ 
- The team should create a method that takes a combination of morse code characters and translate them into a single word. This method should reuse the method that converts a morse code character to a single alphabet. This was not implemented.  ❌
- The team should create a method that translates a sentence of morse code characters into an actual sentence in English. This method should reuse the method that converts a combination of morse code characters into a single word. This was partially implemented because you don't have a method that takes a combination of morse code characters and translates them into a single word. 😄 

#### Hint: Create a method that takes a combination of morse code characters and translates them into a single word. Then reuse that method in the `decode` method. After refactoring, please check that all methods work according to the requirement example given. 👍 😎